### DISCUSSION TITLE
Release 0.7.0

### DISCUSSION THREAD

**Discussion by geriom** [google/docsy#1555] [Announcement]
## IMPORTANT - BREAKING CHANGES

Docsy v0.7.0 is a _significant_ update that includes the (breaking-change) **migration to Bootstrap 5**. For guidance on upgrading Docsy and Bootstrap, see [Migrating to Bootstrap 5.2](https://www.docsy.dev/blog/2023/bootstrap-5-migration/) and the [CHANGELOG](https://github.com/google/docsy/blob/main/CHANGELOG.md#070).

## Release summary

See, https://github.com/google/docsy/blob/main/CHANGELOG.md#070

## Commit details

* Update post-release changelog links by @geriom in https://github.com/google/docsy/pull/1329
* Bump Font-Awesome to latest version 6.2.1 by @deining in https://github.com/google/docsy/pull/1325
* Docsy examples: drop project with private repo by @chalin in https://github.com/google/docsy/pull/1338
* Fix: auto-activation of math and chem blocks by @deining in https://github.com/google/docsy/pull/1341
* Update Netlify instructions by @LisaFC in https://github.com/google/docsy/pull/1342
* Userguide: fixes code example for use of shortcode 'card' by @deining in https://github.com/google/docsy/pull/1347
* Clearer info on customizing i18n strings by @LisaFC in https://github.com/google/docsy/pull/1345
* Add protobuf.dev to examples by @Logofile in https://github.com/google/docsy/pull/1356
* Update to Bootstrap v5: initial work by @deining in https://github.com/google/docsy/pull/517
* Math and chem code clocks, autoactivation: fix use with hugo version < 0.93 by @deining in https://github.com/google/docsy/pull/1351
* [BSv5] More utility updates by @chalin in https://github.com/google/docsy/pull/1361
* [BSv5] YIQ color - final changes by @chalin in https://github.com/google/docsy/pull/1363
* [BSv5] Adjust media-breakpoint-down() argument by @chalin in https://github.com/google/docsy/pull/1367
* Shortcode 'tabpane': minor fix by @deining in https://github.com/google/docsy/pull/1366
* Knative has moved to mkdocs. by @craigbox in https://github.com/google/docsy/pull/1362
* Tab panes: adaptation to Bootstrap 5 by @deining in https://github.com/google/docsy/pull/1371
* [BSv5] Docsy depencendies: bump _rfs.scss to bootstrap version 5.2.3 by @deining in https://github.com/google/docsy/pull/1372
* [BSv5]: make docsy point to latest version of dependencies by @deining in https://github.com/google/docsy/pull/1374
* [BSv5] Footer fixes: reset flex-shrink, and more by @chalin in https://github.com/google/docsy/pull/1373
* [BSv5] Script for persisting tabpanes: Fix by @deining in https://github.com/google/docsy/pull/1378
* Finnish translations by @jannepeltola in https://github.com/google/docsy/pull/1379
* [BSv5] replace lighten() by tint-color(), and var/func clean up by @chalin in https://github.com/google/docsy/pull/1384
* [BSv5] Fix SCSS functions import issue, and replace darken() by @chalin in https://github.com/google/docsy/pull/1388
* go.sum: clear unused entries, and tabpane cleanup by @deining in https://github.com/google/docsy/pull/1386
* Copyedits to installation-prerequisites.md by @chalin in https://github.com/google/docsy/pull/1390
* Shortcode imgproc: add crop command by @deining in https://github.com/google/docsy/pull/1272
* Upgrade to Hugo 0.110.0 and normalize script names by @chalin in https://github.com/google/docsy/pull/1402
* Upgrade FontAwesome to 6.3.0 and tweak update script by @chalin in https://github.com/google/docsy/pull/1403
* Rename config.yaml to hugo.yaml by @chalin in https://github.com/google/docsy/pull/1404
* [BSv5][JS]: fix dropdowns and more by @deining in https://github.com/google/docsy/pull/1382
* [BSv5] Drop invalid and unnecessary tooltip attribute by @chalin in https://github.com/google/docsy/pull/1407
* Website, section 'About': fix broken link by @deining in https://github.com/google/docsy/pull/1411
* User guide: rename config file base name to 'hugo' (#1405) by @deining in https://github.com/google/docsy/pull/1413
* [BS5] Breadcrumbs: active-breadcrumb fix by @chalin in https://github.com/google/docsy/pull/1414
* [BS5] Adjust HTML of dropdowns for version and lang by @chalin in https://github.com/google/docsy/pull/1415
* Tabbed pane, fix: do not print blank first line by @deining in https://github.com/google/docsy/pull/1416
* Bump mermaid and KaTeX to their latest versions by @deining in https://github.com/google/docsy/pull/1408
* Module setup: bump FontAwesome to latest version 6.3.0 by @deining in https://github.com/google/docsy/pull/1409
* [BSv5] Offline-search adjust arrow class name by @chalin in https://github.com/google/docsy/pull/1427
* [BSv5] Navbar: add inner container + cleanup by @chalin in https://github.com/google/docsy/pull/1422
* [BSv5] Adjusted offline search popover by @tekezo in https://github.com/google/docsy/pull/1429
* Search resource cleanup and style tweak by @chalin in https://github.com/google/docsy/pull/1434
* Fix community section colors by @chalin in https://github.com/google/docsy/pull/1437
* package.json: add license and homepage by @deining in https://github.com/google/docsy/pull/1428
* Navigation: one more config-to-hugo file rename by @chalin in https://github.com/google/docsy/pull/1424
* NPM package config: set Node version constraint by @chalin in https://github.com/google/docsy/pull/1440
* [BSv5] Drop import of rtl/main by @chalin in https://github.com/google/docsy/pull/1443
* tabpane warning: make it easier to find source page by @chalin in https://github.com/google/docsy/pull/1448
* Pin Docsy version in other installation options by @chalin in https://github.com/google/docsy/pull/1439
* Use <pre> tag for mermaid code blocks by @deining in https://github.com/google/docsy/pull/1451
* [BSv5] Drop redundant setting of BS `$enable-rounded` variable by @chalin in https://github.com/google/docsy/pull/1452
* Reformat SCSS sources using Prettier by @chalin in https://github.com/google/docsy/pull/1454
* [BSv5] Drop `bg-gradient-variant()` mixin by @chalin in https://github.com/google/docsy/pull/1455
* Delete duplicate mixin definitions by @chalin in https://github.com/google/docsy/pull/1458
* [BSv5] Drop display SASS variables that no longer exist by @chalin in https://github.com/google/docsy/pull/1459
* [BSv5] Docsy variables cleanup and BS version test by @chalin in https://github.com/google/docsy/pull/1462
* [BSv5] blocks/cover row-col hierarchy fix by @chalin in https://github.com/google/docsy/pull/1467
* shortcode 'blocks/lead' improvements by @deining in https://github.com/google/docsy/pull/1398
* [BSv5] blocks/lead row-col hierarchy fix by @chalin in https://github.com/google/docsy/pull/1469
* [BSv5] community/list row-col hierarchy fix by @chalin in https://github.com/google/docsy/pull/1471
* Avoid warning 'calling IsSet with unsupported type "invalid" (<nil>)' by @deining in https://github.com/google/docsy/pull/1465
* [BSv5] blocks/section row-col hierarchy fix by @chalin in https://github.com/google/docsy/pull/1472
* [BSv5] Factor out Docy's table style and set max-width by @chalin in https://github.com/google/docsy/pull/1474
* [BSv5] Set tables to display:block and explain why by @chalin in https://github.com/google/docsy/pull/1477
* [BSv5] Fix position of blog's RSS button by @chalin in https://github.com/google/docsy/pull/1480
* Shortcodes: Update readfile to throw compile error when file not found by @aimeeu in https://github.com/google/docsy/pull/1457
* Drop mycplus.com iframe from Shortcode page by @chalin in https://github.com/google/docsy/pull/1475
* [BSv5] Adapt cardpane shortcode by @deining in https://github.com/google/docsy/pull/1376
* Tabpane shortcodes: improved reporting of warnings and errors by @deining in https://github.com/google/docsy/pull/1478
* module: ignore Bootstrap config by @razonyang in https://github.com/google/docsy/pull/1509
* Update CONTRIBUTING.md by @emckean in https://github.com/google/docsy/pull/1511
* User guide: fix warning about deprecated language config by @deining in https://github.com/google/docsy/pull/1519
* [BSv5] fix horizontal scrollbar issues by @yann-soubeyrand in https://github.com/google/docsy/pull/1502
* [BSv5] Fix RSS button style by @chalin in https://github.com/google/docsy/pull/1521
* [BSv5] fix main container height by @yann-soubeyrand in https://github.com/google/docsy/pull/1501
* [BSv5] Footer: drop flex-shrink tweak + other adjustments by @chalin in https://github.com/google/docsy/pull/1523
* Minify hugo-served pages by @chalin in https://github.com/google/docsy/pull/1527
* [BSv5] Refactor Click-to-copy by @geriom in https://github.com/google/docsy/pull/1447
* Fix: shortcode `imgproc` does not work with % for markdown body by @deining in https://github.com/google/docsy/pull/1522
* Adjust Mermaid block styling and block content by @deining in https://github.com/google/docsy/pull/1456
* Make automatic detection of draw.io diagrams an opt-in by @deining in https://github.com/google/docsy/pull/1470
* Fix link text and hover color contrast for all bgs by @chalin in https://github.com/google/docsy/pull/1525
* Bump dependency Font-Awesome to latest version 6.4.0 by @deining in https://github.com/google/docsy/pull/1529
* Revert update of Hugo version by @chalin in https://github.com/google/docsy/pull/1530
* package.json: rename cSpell:ignore entry tag to "spelling" by @chalin in https://github.com/google/docsy/pull/1536
* Blog on bootstrap-5 migration by @chalin in https://github.com/google/docsy/pull/1535
* Make docsy theme point to latest version of dependencies by @deining in https://github.com/google/docsy/pull/1533
* Upgrade Hugo to 0.112.3 and fix `en` title by @deining in https://github.com/google/docsy/pull/1534
* Fix site title post Hugo upgrade by @chalin in https://github.com/google/docsy/pull/1539
* Keep lang param DRY by @chalin in https://github.com/google/docsy/pull/1543
* Add scripts & upgrade Hugo to 0.113.0, autoprefixer, and postcss-cli by @chalin in https://github.com/google/docsy/pull/1544
* CHANGELOG: add min hugo version by @chalin in https://github.com/google/docsy/pull/1546
* Click-to-copy: remove vertical space from button div by @chalin in https://github.com/google/docsy/pull/1549
* [BSv5] Footer link icons: use text color by @chalin in https://github.com/google/docsy/pull/1547
* CHANGELOG: update 0.4.0 submodule cleanup instructions by @chalin in https://github.com/google/docsy/pull/1550
* Add top-level NPM test script by @chalin in https://github.com/google/docsy/pull/1551
* Correct required hugo version (#1546) by @deining in https://github.com/google/docsy/pull/1552
* Fix: correct min_version in theme.toml by @deining in https://github.com/google/docsy/pull/1553
* Release v0.7.0 preparation by @geriom in https://github.com/google/docsy/pull/1554

**Full Changelog**: https://github.com/google/docsy/compare/v0.6.0...v0.7.0

<hr /><em>This discussion was created from the release <a href='https://github.com/google/docsy/releases/tag/v0.7.0'>0.7.0</a>.</em>

**Comment by jmichelgarcia** [comment]
Sweet! 🚀 🚀 🚀 

After upgrading I'm facing 
`Docsy requires Bootstrap v5, but v4 was detected. Did you forget to update Docsy dependencies?`

Yet `go.sum` says

```
github.com/FortAwesome/Font-Awesome v0.0.0-20230327165841-0698449d50f2/go.mod h1:IUgezN/MFpCDIlFezw3L8j83oeiIuYoj28Miwr/KUYo=
github.com/google/docsy v0.7.0 h1:JaeZ0/KufX/BJ3SyATb/fmZa1DFI7o5d9KU+i6+lLJY=
github.com/google/docsy v0.7.0/go.mod h1:5WhIFchr5BfH6agjcInhpLRz7U7map0bcmKSpcrg6BE=
github.com/google/docsy/dependencies v0.7.0 h1:/xUlWCZOSMDubHfrhIz1YtaRn2Oc/swfJ7OUfglXE8U=
github.com/google/docsy/dependencies v0.7.0/go.mod h1:gihhs5gmgeO+wuoay4FwOzob+jYJVyQbNaQOh788lD4=
github.com/twbs/bootstrap v5.2.3+incompatible/go.mod h1:fZTSrkpSf0/HkL0IIJzvVspTt1r9zuf7XlZau8kpcY0=
```

Will have a deeper look later!

**Reply by chalin** [reply]
Hi @jmichelgarcia - I usually see that error when Docsy is used via a Git submodule or NPM package. If that's how you are bring in Docsy, have you re-run `npm install`?

If you are using Hugo modules, we'll need more information about the site you're trying to upgrade. We have sites building fine under Hugo modules and Docsy 0.7.0.

/cc @deining

**Reply by marcelbaumann** [reply]
Hello

I upgraded to 0.7.0 today. I use the git clone variant _git clone --branch v0.7.0 https://github.com/google/docsy.git_.
So this variant seems to work without problems. The commands I use are:

_cd themes_
_git clone --branch v0.7.0 https://github.com/google/docsy.git_
_cd docsy_
_npm install_

_cd $siteDir_
_npm install -D autoprefixer_
_npm install -D postcss_
_npm install -D postcss-cli__

The upgrade process worked without any problem and the [new generated site](https://blog.tangly.net/) is working as expected.

Thanks for the new release.

**Reply by chalin** [reply]
Nice website @marcelbaumann! Thanks for sharing about your smooth upgrade experience. Would you be willing to submit a PR to add your site to our [Examples](https://www.docsy.dev/docs/examples/) page?

**Reply by jmichelgarcia** [reply]
**go.mod**
```
module github.com/xxx/docs

go 1.19

require (
	github.com/google/docsy v0.7.0 // indirect
	github.com/google/docsy/dependencies v0.7.0 // indirect
)
```

**package.json**
```
"devDependencies": {
    "autoprefixer": "^10.4.14",
    "postcss": "^8.4.24",
    "postcss-cli": "^10.1.0",
    "hugo-extended": "0.113.0"
  }
```

**hugo.toml**
```
[module]
  proxy = "direct"
  # uncomment line below for temporary local development of module
  # replacements = "github.com/google/docsy -> ../../docsy"
  [module.hugoVersion]
    extended = true
    min = "0.113.0"
  [[module.imports]]
    path = "github.com/google/docsy"
    disable = false
  [[module.imports]]
    path = "github.com/google/docsy/dependencies"
    disable = false
```
---
I ran

- `npm cache clean`
- `npm install`
- `npm run clean (which runs hugo mod tidy)`
- `npm run build`

I see a bunch of these errors

```
Start building sites … 
hugo v0.113.0-085c1b3d614e23d218ebf9daad909deaa2390c9a+extended darwin/arm64 BuildDate=2023-06-05T15:04:51Z VendorInfo=gohugoio
WARNING: Found no color leading to 4.5:1 contrast ratio against #EE635E...
         on line 168:3 of ../../../../var/folders/cn/myqbxp2d78d_mqqj9fj0nkh00000gp/T/hugo_cache/modules/filecache/modules/pkg/mod/github.com/twbs/bootstrap@v5.2.3+incompatible/scss/_functions.scss, in function `color-contrast`
         from line 21:19 of ../../../../var/folders/cn/myqbxp2d78d_mqqj9fj0nkh00000gp/T/hugo_cache/modules/filecache/modules/pkg/mod/github.com/twbs/bootstrap@v5.2.3+incompatible/scss/mixins/_forms.scss, in mixin `form-validation-state`
         from line 10:12 of ../../../../var/folders/cn/myqbxp2d78d_mqqj9fj0nkh00000gp/T/hugo_cache/modules/filecache/modules/pkg/mod/github.com/twbs/bootstrap@v5.2.3+incompatible/scss/forms/_validation.scss
         from line 9:9 of ../../../../var/folders/cn/myqbxp2d78d_mqqj9fj0nkh00000gp/T/hugo_cache/modules/filecache/modules/pkg/mod/github.com/twbs/bootstrap@v5.2.3+incompatible/scss/_forms.scss
         from line 21:9 of ../../../../var/folders/cn/myqbxp2d78d_mqqj9fj0nkh00000gp/T/hugo_cache/modules/filecache/modules/pkg/mod/github.com/twbs/bootstrap@v5.2.3+incompatible/scss/bootstrap.scss
         from line 9:9 of stdin
```

and then

```
Total in 7988 ms
Error: error building site: TOCSS: failed to transform "scss/main.scss" (text/x-scss): "/var/folders/cn/myqbxp2d78d_mqqj9fj0nkh00000gp/T/hugo_cache/modules/filecache/modules/pkg/mod/github.com/google/docsy@v0.7.0/assets/scss/main.scss:13:3": Docsy requires Bootstrap v5, but v4 was detected. Did you forget to update Docsy dependencies?
```

**Reply by deining** [reply]
> ```
> Total in 7988 ms
> Error: error building site: TOCSS: failed to transform "scss/main.scss" (text/x-scss): "/var/folders/cn/myqbxp2d78d_mqqj9fj0nkh00000gp/T/hugo_cache/modules/filecache/modules/pkg/mod/github.com/google/docsy@v0.7.0/assets/scss/main.scss:13:3": Docsy requires Bootstrap v5, but v4 was detected. Did you forget to update Docsy dependencies?
> ```

This resembles #1556, therefore my question: are you defining your own, custom `assets/scss/main.css`? If so, this is not considered best practice and can cause problems. You are using bootstrap 5.2.3 for sure, my guess is, that some custom overwrite from your project causes your problems. Can you share your site repo so that we can have a look?

If you run `hugo config mounts` at the console, what is the result?

**Reply by jmichelgarcia** [reply]
> are you defining your own, custom assets/scss/main.css

No.

> If you run hugo config mounts at the console, what is the result?

```
{
   "path": "github.com/xxx/docs",
   "version": "",
   "time": "0001-01-01T00:00:00Z",
   "owner": "",
   "dir": "/Users/jm/dev/docs",
   "mounts": [
      {
         "source": "package.json",
         "target": "assets/_jsconfig/package.json"
      },
      {
         "source": "content/en",
         "target": "content",
         "lang": "en"
      },
      {
         "source": "content/fr",
         "target": "content",
         "lang": "fr"
      },
      {
         "source": "data",
         "target": "data"
      },
      {
         "source": "layouts",
         "target": "layouts"
      },
      {
         "source": "i18n",
         "target": "i18n"
      },
      {
         "source": "archetypes",
         "target": "archetypes"
      },
      {
         "source": "assets",
         "target": "assets"
      },
      {
         "source": "static",
         "target": "static"
      },
      {
         "source": "static_fr",
         "target": "static"
      }
   ]
}
{
   "path": "github.com/google/docsy",
   "version": "v0.7.0",
   "time": "2023-06-08T17:00:26Z",
   "owner": "github.com/xxx/docs",
   "dir": "/var/folders/cn/myqbxp2d78d_mqqj9fj0nkh00000gp/T/hugo_cache/modules/filecache/modules/pkg/mod/github.com/google/docsy@v0.7.0/",
   "mounts": [
      {
         "source": "assets",
         "target": "assets"
      },
      {
         "source": "i18n",
         "target": "i18n"
      },
      {
         "source": "layouts",
         "target": "layouts"
      },
      {
         "source": "static",
         "target": "static"
      },
      {
         "source": "package.json",
         "target": "assets/_jsconfig/package.json"
      },
      {
         "source": "postcss.config.js",
         "target": "assets/_jsconfig/postcss.config.js"
      }
   ]
}
{
   "path": "github.com/google/docsy/dependencies",
   "version": "v0.7.0",
   "time": "2023-06-08T17:00:26Z",
   "owner": "github.com/xxx/docs",
   "dir": "/var/folders/cn/myqbxp2d78d_mqqj9fj0nkh00000gp/T/hugo_cache/modules/filecache/modules/pkg/mod/github.com/google/docsy/dependencies@v0.7.0/",
   "mounts": [
      {
         "source": "assets/bootstrap/scss/_vendor",
         "target": "assets/vendor/bootstrap/scss/vendor"
      },
      {
         "source": "assets",
         "target": "assets"
      }
   ]
}
{
   "path": "github.com/twbs/bootstrap",
   "version": "v5.2.3+incompatible",
   "time": "2022-11-21T18:19:01Z",
   "owner": "github.com/google/docsy/dependencies",
   "dir": "/var/folders/cn/myqbxp2d78d_mqqj9fj0nkh00000gp/T/hugo_cache/modules/filecache/modules/pkg/mod/github.com/twbs/bootstrap@v5.2.3+incompatible/",
   "mounts": [
      {
         "source": "scss",
         "target": "assets/vendor/bootstrap/scss"
      },
      {
         "source": "dist/js",
         "target": "assets/vendor/bootstrap/dist/js"
      },
      {
         "source": "package.json",
         "target": "assets/_jsconfig/package.json"
      }
   ]
}
{
   "path": "github.com/FortAwesome/Font-Awesome",
   "version": "v0.0.0-20230327165841-0698449d50f2",
   "time": "2023-03-27T16:58:41Z",
   "owner": "github.com/google/docsy/dependencies",
   "dir": "/var/folders/cn/myqbxp2d78d_mqqj9fj0nkh00000gp/T/hugo_cache/modules/filecache/modules/pkg/mod/github.com/!fort!awesome/!font-!awesome@v0.0.0-20230327165841-0698449d50f2/",
   "mounts": [
      {
         "source": "scss",
         "target": "assets/vendor/Font-Awesome/scss"
      },
      {
         "source": "webfonts",
         "target": "static/webfonts"
      }
   ]
}
```

**Comment by deining** [comment]
Your mount points look all good. Hard to tell what's going wrong here without seeing your code.

What you can do:

```
hugo mod clean
hugo mod vendor
hugo server
```

Does this help?

If not, try to identify the offending file by removing one folder after the other from your site repo:

```
rm -rf _vendor
hugo server
rm -rf assets
hugo server
rm -rf content
hugo server
...
```

Or provide access to your repo (or if you can't strip down your repo and remove all sensitive information you do not want to share).

**Reply by jmichelgarcia** [reply]
It was linked to this PR https://github.com/google/docsy/pull/1459/files
I had my own definition of `Display Styles` in `_variables_project.scss`

**Reply by chalin** [reply]
Thanks for debugging that @jmichelgarcia, and apologies for the false v4 positive. I've created the following issue, and have prepared a better test:

- #1574

**Reply by chalin** [reply]
FYI, there's a new test in place. For details, see #1579.

**Comment by danieltikamori** [comment]
Hello everyone! I finally used Hugo and choose the Docsy theme. Add the theme as a Hugo Module. But I'm having the following error when run `hugo server`:

`content/en/_index.md:5:1": failed to extract shortcode: template for shortcode "blocks/cover" not found
`

Researched for hours the solution, none of them worked, can someone help me? I'm using the original files just to test. The from scratch version works fine, but I want the prepopulated site.

**Comment by deining** [comment]
> Hello everyone! I finally used Hugo and choose the Docsy theme. Add the theme as a Hugo Module. But I'm having the following error when run `hugo server`:
> 
> `content/en/_index.md:5:1": failed to extract shortcode: template for shortcode "blocks/cover" not found `
> 
> Researched for hours the solution, none of them worked, can someone help me? I'm using the original files just to test. The from scratch version works fine, but I want the prepopulated site.

Try

``
hugo mod clean --all
``

If doesn't help have a look at the other solutions offered [here](https://github.com/google/docsy-example/issues/41).

**Reply by danieltikamori** [reply]
Thank you @deining, but I've already tried many times these solutions and didn't work. Also tried to create my own templates, but only created new errors. I'm going to give up as it is wasting hours of my short time.

**Comment by deining** [comment]
> Thank you @deining, but I've already tried many times these solutions and didn't work. Also tried to create my own templates, but only created new errors. I'm going to give up as it is wasting hours of my short time.

Sorry to hear that. Without more information it is impossible to find out what's going wrong here. Is your site publicly available or can you grant access to it? This would be really, really helpful!

**Reply by danieltikamori** [reply]
It is still in development stage, did the steps of Option 2: Using the GitHub UI (local copy + associated GitHub repo) and tried to preview the site through `hugo server`.

**Comment by deining** [comment]
> It is still in development stage, did the steps of Option 2: Using the GitHub UI (local copy + associated GitHub repo) and tried to preview the site through `hugo server`.

I just tried on a linux box, and it worked perfectly fine for me. What versions of hugo and go are you using? What is your OS? What does `hugo mod graph` tell you?

**Reply by danieltikamori** [reply]
Using Linux
Go version: go version go1.20.5 linux/amd64
Hugo version: hugo v0.101.0+extended linux/amd64 BuildDate=unknown
hugo mod graph: shows nothing.

The repo: https://github.com/danieltikamori/danielblog

**Reply by danieltikamori** [reply]
Asked to ChatGPT, Codeium, Bard, etc and they only tell to create my own cover template.

**Comment by deining** [comment]
> Using Linux Go version: go version go1.20.5 linux/amd64 Hugo version: hugo v0.101.0+extended linux/amd64 BuildDate=unknown hugo mod graph: shows nothing.

```
hugo v0.101.0+extended linux/amd64
```

Bingo.
From the [changelog](https://github.com/google/docsy/blob/main/CHANGELOG.md):

```
[0.7.0]

Breaking changes:

Hugo release 0.110.0 or later is required.
```

Upgrade hugo and the site will run smoothly. If you can't upgrade hugo, in the site root rename your `hugo.toml` to `config.yaml`, and it will work, too. Please note that this is **not** recommended, however.

**Reply by danieltikamori** [reply]
Worked!!! Thank you very much!
My distro package manager upgrades until hugo v0.101.0+extended
Had to do manually the upgrade.

**Comment by deining** [comment]
> Worked!!! Thank you very much!

Glad I could help out here! Happy authoring with Docsy!
The current behaviour in case of older hugo versions  is suboptimal. I created #1591 to address this deficiency. I hope we will be able to come up with a remedy soon.
